_dev/src/components/commons/banner-campaigns.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <b-card class="shadow-sm border-0 mb-3">
+    <b-card-body>
+      <div class="d-flex align-items-center">
+        <img
+          src="@/assets/images/Google-Commercial-icon.svg"
+          width="75"
+          height="75"
+          alt=""
+        >
+        <span class="mr-3 ml-3">
```

Comment:
```suggestion
        <div class="mr-3 ml-3">
```
It's not allowed to have a `h3` in a `span`.
A heading tag belongs to the [Flow content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#flow_content) category, but `span` tag allows only phrasing content.

*span* | Permitted content | [Phrasing content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#phrasing_content).
-- | -- | --
***h3*** | **[Content categories](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories)** | *[Flow content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#flow_content), heading content, palpable content.*

Reply:
thanks !